frontend/src/metabase/visualizations/visualizations/Text/Text.jsx

```diff
@@ -75,6 +77,10 @@ export function Text({
     [dashcard, dashboard, parameterValues, translatedText],
   );
 
+  const updateParameterMappings = useCallback(() => {
+    dispatch(updateParameterMappingsForDashcardText(dashcard.id));
+  }, [dashcard.id, dispatch]);
+
```

Comment:
Are we fine to dispatch it directly from the Text component? 
I see that it's related to a dashcard and it already gets via `useSelector` the `parameterValues` from the state, so i assume that we can also update `parameter_mappings` from it